files-community/Files
# Feature: Added new icons to the properties page

**Resolved / Related Issues**
   Closes #issue...

**Validation**
How did you test these changes?
   1. Open the properties page for folders
   2. Open the properties page for files
   3. Open the properties page for exe files

**Screenshots (optional)**
Add screenshots here.
